src/main/java/dk/dbc/opensearch/OpensearchQuery.java

```diff
@@ -141,9 +153,14 @@ public String build() throws java.io.UnsupportedEncodingException {
         if(this.bc != null && !this.bc.isBlank()) {
             query = addToQueryString(query, "bc=" + bc);
         }
+        if(this.marc001b != null && !this.marc001b.isBlank()) {
+            query = addToQueryString(query, "marc.001b=" + marc001b);
+        }
 
         // Encode the query, but convert encoded blankspace from %2B (+) to %20 (real blank)
```

Comment:
Just curious - hvordan laver man så : 
(marc.001b=870970 OR marc.001b=870971 OR marc.001b=870979 OR marc.001b=190004) AND (lti="tadiff a flexible c++*")
I katten giver det hit på 21916749.